panel/src/components/Forms/Input/TextInput.vue

```diff
@@ -87,6 +87,9 @@ export default {
   computed: {
     direction() {
       return direction(this);
+    },
+    placeholderLabel() {
+      return this.placeholder ? this.placeholder.toString(this) : null;
```

Comment:
The `placeholder` prop in this component is defined as `String` only. Have I misunderstood something?